### DISCUSSION TITLE
Color-coded Dropdown options

### DISCUSSION THREAD

**Discussion by explorigin** [community/community#6615] [Projects and Issues] [answered]
Allow us to assign colors to dropdown options to make them more visually distinct in a list.

**Comment by nicbarajas** [comment, marked as answer]
This may be similar to #6234 which already has a bunch of upvotes!

**Reply by j0siepy** [reply]
Thank you so much for your patience here! We've shipped this change out today! 

https://user-images.githubusercontent.com/98360703/217907150-6be312a2-952e-4747-9bf8-f8a61cdce6ee.mov
